docs/contracts/uniswapx/guides/arbitrumfiller.md

```diff
@@ -0,0 +1,60 @@
+---
+id: arbitrumfiller
+title: Arbitrum Pilot
+sidebar_position: 1
+---
+
+# Arbitrum Pilot Overview
+Starting June 19 2024, the Uniswap team will be piloting running a portion of trades on Arbitrum through UniswapX. Unlike UniswapX on mainnet, these orders will have **no RFQ portion and thus no exclusivity** during the pilot. 
+
+Filling on Arbitrum, however, follows the same two steps as filling on Mainnet: 1) Retrieving signed orders and 2) Filling orders. 
+
+## Retrieving Signed Orders
+All signed Dutch Orders on Arbitrum, created through the Uniswap UI will be available via the UniswapX Orders Endpoint. We have [swagger documentation](https://api.uniswap.org/v2/uniswapx/docs) but see below for a quick example curl.
+
+```
+GET https://api.uniswap.org/v2/orders?orderStatus=open&chainId=42161&limit=1000
+```
+
+Use the [UniswapX SDK](https://github.com/Uniswap/sdks/tree/main/sdks/uniswapx-sdk) to parse the `encodedOrder` field returned from endpoint. Each one of these `Orders` represents a fillable user trader. 
```

Comment:
Can you mention that we also support the webhook for Arbitrum orders and link to the `Signed Order Webhook Notifications` section in the other doc?